# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mesa/rusticl-enabled-devices.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- mesa-24.3.4/src/gallium/frontends/rusticl/mesa/pipe/device.rs	2025-01-22 19:12:23.000000000 +0100
+++ mesa-24.3.4/src/gallium/frontends/rusticl/mesa/pipe/device.rs.orig	2025-02-04 09:18:55.990638013 +0100
@@ -59,6 +59,8 @@
     let default_devs: &[&str] = &[
         #[cfg(any(rusticl_enable_asahi, rusticl_enable_auto))]
         "asahi",
+
+        "iris","llvmpipe","nouveau","panfrost","radeonsi","r600"
     ];
 
     // I wished we could use different iterators, but that's not really working out.
